src/tests/work-item/work-item-list/work-item-detail.page.js

```diff
@@ -80,9 +80,35 @@ WorkItemDetailPage.prototype  = Object.create({}, {
   },
 
   /* Icon edit buttons */
-  workItemTitleEditIcon:  {
+  editHover:   {
     get: function ()
-    { return element(by.id("workItemTitle_btn_edit")); }
+    { 
+      return element(by.id("")); 
```

Comment:
Does this ID need to reference something? (Or maybe I just don't understand how it works.)